hadoop-ozone/dist/src/main/compose/xcompat/test.sh

```diff
@@ -50,6 +50,7 @@ _kinit() {
 }
 
 _init() {
+  container=scm
```

Comment:
Why does this method need to explicitly specify this, while other's leave the container variable up to the caller?

Reply:
Others are executed with `new_client` and `old_client` in turn (which define the container), this one needs to be done only once.